monitoring/prometheus/alerts/ceph_default_alerts.yml

```
@@ -9,7 +9,10 @@ groups:
           type: ceph_default
           oid: 1.3.6.1.4.1.50495.15.1.2.2.1
         annotations:
-          description: Ceph in health_error state for more than 5m.
+          description: >
+            Ceph in HEALTH_ERROR state for more than 5 minutes.
+            Please check the status of OSDs and pools.
```

```suggestion
            Please check "ceph health detail" for more information.
```
Other things besides OSDs and pools can cause health issues.